repos/fdbt-site/src/utils/apiUtils/matching.ts

```diff
@@ -83,6 +104,9 @@ export const getMatchingFareZonesAndUnassignedStopsFromForm = (
 export const isFareStageUnassigned = (userFareStages: UserFareStages, matchingFareZones: MatchingFareZones): boolean =>
     userFareStages.fareStages.some((stage) => !matchingFareZones[stage.stageName]);
 
+export const isFareStageUnassignedEditTicket = (fareStages: string[], matchingFareZones: MatchingFareZones): boolean =>
```

Comment:
The name of this function doesnt seem right - what is the code doing?

Reply:
it is the same function as isFareStageUnassigned